config/configstore/model_test.go

```diff
@@ -400,7 +400,7 @@ func createMockConfig() *mockConfig {
 	c.On("GetNumWorkers").Return(2).Once()
 	c.On("GetWorkerWaitTimeMS").Return(1).Once()
 	c.On("GetEthereumNodeURL").Return("dummyNode").Once()
-	c.On("GetIdentityID").Return(utils.RandomSlice(6), nil).Once()
+	c.On("GetIdentityID").Return(utils.RandomSlice(common.AddressLength), nil).Once()
```

Comment:
Please wrap `common.AddressLength` (or any other things from ethereum.common) when using outside identity. We should avoid past mistakes of leaky abstractions.

Reply:
I had it already in mind to add a length constant to identity package. I will fix it on monday

Reply:
Added required changes.